t/food.t

```diff
@@ -457,4 +457,122 @@ foreach (@{$nutriments_tables{europe}}) {
 	ok(exists_taxonomy_tag("nutrients", $tagid), "$tagid exists in the nutrients taxonomy");
 }
 
+# Test normalize_nutriment_value_and_modifier()
+# and assign_nid_modifier_value_and_unit()
+
```

Comment:
I think you could add more comments before each test,
it will help distinguish test, but also have an idea of what you are testing exactly.

I'll do some suggestions.

Reply:
Perfect, thank you!